vulnerabilities/tests/test_npm.py

```diff
@@ -23,7 +23,7 @@
 
 
 from django.test import TestCase
-from vulnerabilities.scraper.npm import remove_spaces, get_all_version, extract_data
+from vulnerabilities.scraper.npm import remove_spaces, get_all_versions, extract_data
```

Comment:
I personally prefer having each import on its own line.
Also  separate imports in four sorted group with an empty line between each: future import, standard lib, thirdparty, own modules. Would this work for you too?

Reply:
yes